config/serverless.oblt.yml

```diff
@@ -99,30 +98,27 @@ xpack.features.overrides:
     privileges:
       # SLOs `All` feature privilege should implicitly grant `All` access to Observability app.
       all.composedOf:
-        - feature: "observability"
-          privileges: [ "all" ]
+        - feature: 'observability'
+          privileges: ['all']
       # SLOs `Read` feature privilege should implicitly grant `Read` access to Observability app.
       read.composedOf:
-        - feature: "observability"
-          privileges: [ "read" ]
+        - feature: 'observability'
+          privileges: ['read']
   ### Stack alerts is hidden in Role management since it's not needed.
   stackAlerts.hidden: true
   ### Synthetics feature privileges are fine-tuned to grant access to Observability app.
   uptime:
     ### By default, this feature named as `Synthetics and Uptime`, but should be renamed to `Synthetics` since `Uptime` is not available.
-    name: "Synthetics"
+    name: 'Synthetics'
     privileges:
       # Synthetics `All` feature privilege should implicitly grant `All` access to Observability app.
       all.composedOf:
-        - feature: "observability"
-          privileges: [ "all" ]
+        - feature: 'observability'
+          privileges: ['all']
       # Synthetics `Read` feature privilege should implicitly grant `Read` access to Observability app.
       read.composedOf:
-        - feature: "observability"
-          privileges: [ "read" ]
-
-## Enable the slo plugin
-xpack.slo.enabled: true
```

Comment:
💬 Only removed this feature flag that got removed from our config (was set on defaultValue: true for awhile now, and never used in code).